package.json

```diff
@@ -12,10 +12,13 @@
     "elm-webpack-loader": "^4.4.0",
     "file-loader": "^1.1.5",
     "node-sass": "^4.7.2",
-    "sass-loader": "^6.0.6",
-    "style-loader": "^0.19.0",
-    "url-loader": "^0.6.2",
-    "webpack": "^3.10.0",
-    "webpack-dev-server": "^2.9.7"
+    "sass-loader": "^7.0.1",
+    "style-loader": "^0.21.0",
+    "url-loader": "^1.0.1",
+    "webpack": "^4.6.0",
+    "webpack-dev-server": "^3.1.3"
+  },
+  "devDependencies": {
+    "webpack-cli": "^2.1.2"
```

Comment:
Do we need `webpack-cli`? I think it's fine to remove it.

Reply:
We might not, when installing webpack the installation prompted me if it should be installed. I'll test it without!

Reply:
Now that I think of it, probably all of the deps in `dependencies` can go into `devDependencies`, since they're all build tools

Reply:
True, haha, let's update it as well.